tests/Resources/ti.ui.textfield.addontest.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti, Titanium */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
+
+describe('Titanium.UI.TextField', function () {
+	var win;
+
+	afterEach(function () {
+		if (win) {
+			win.close();
+		}
+		win = null;
+	});
+
+	it.ios('hasText', function (finish) {
+		win = Ti.UI.createWindow();
+		 
+		var textFieldA = Ti.UI.createTextField({
+		  top: '60dip',
+		  value: 0
+		});
+		 
+		win.add(textFieldA);
+		 
```

Comment:
<!--
  2 failure:  tests/Resources/t..., tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.ui.textfield.addontest.js;
  Line: 33;
-->
- :no_entry_sign: tests/Resources/ti.ui.textfield.addontest.js line 33 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
- :no_entry_sign: tests/Resources/ti.ui.textfield.addontest.js line 33 – Trailing spaces not allowed. (no-trailing-spaces)